src/scenekit.cs

```diff
@@ -3894,6 +4080,13 @@ interface SCNTechnique : SCNAnimatable, NSCopying, NSSecureCoding {
 		[iOS (9,0)][Mac (10,11)]
 		[Internal, Export ("setObject:forKeyedSubscript:")]
 		void _SetObject ([NullAllowed] NSObject obj, INSCopying key);
+
+#if XAMCORE_2_0 // Metal is 64 bit only
```

Comment:
Everything is `XAMCORE_2_0` now, so this `#if` is not needed.

Reply:
True it's redundant but the PR is mergeable right now so I'm not gonna rebuild it (:
Will avoid next time.